### PR TITLE
intial support for .ico generation

### PR DESCRIPTION
**Pull request by leaanthony**
This will generate a windows .ico file based on the PNG file "appicon.png". It will resize the image to 256x256 and generate an .ico file with that single image in it.

### CODE REVIEW

File: cmd/package.go [hunk 1]
```diff
@@ -1,9 +1,12 @@
-package cmd
+0package cmd
```
[the remark this answers is not in the document]
Comment (tmclane) [reply cmd/package.go:1]: BUSTED!

**Comment by leaanthony** [comment]
FAT FINGERED

**Comment by tmclane** [comment]
Squash those extras!

**Comment by dedo1911** [comment]
Tried. Works as expected. Thanks!

**Pull request merged**